src/EPR.RegulatorService.Frontend.Web/Controllers/Applications/ApplicationsController.cs

```diff
@@ -94,7 +105,7 @@ public async Task<IActionResult> Applications(ApplicationsRequestViewModel viewM
             {
                 if (organisationId == null)
                 {
-                    _logger.LogError($"Organisation id was null.");
+                    _logOrganisationIdNull(_logger, null);
```

Comment:
not really sure what is going on here

Reply:
@hardik-desai-kcl  SQ complaining as on this line we need use LoggerMessage.

_logger.LogError($"Organisation id was null.");
For improved performance, use the LoggerMessage delegates instead of calling 'LoggerExtensions.LogError(ILogger, string?, params object?[])'

Updated the code to fix nothing major change it's just way of defining loggers.

Reply:
Try this
                    _logger.LogError("Organisation id was null.");